services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/search/ChangeMessageToSearchDocumentTranslator.java

```diff
@@ -30,7 +30,7 @@ public class ChangeMessageToSearchDocumentTranslator {
 	@Autowired
 	CloudSearchLogger recordLogger;
 
-	Document generateSearchDocumentIfNecessary(ChangeMessage change) {
+	public Document generateSearchDocumentIfNecessary(ChangeMessage change) {
```

Comment:
We can have a simplified version of this without the cloudsearch specific logging for the purpose of the mvp.